### PR TITLE
Tier Config Requests

### PR DESCRIPTION
**Pull request by JaviCerveraIngram**
Support for the tier/config-requests API through the TierConfigRequestAutomation class.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ $ pip install connect-sdk
 
 ### Example
 ```python
-from connect import FulfillmentAutomation
+from connect import FulfillmentAutomation, TierConfigRequestAutomation
```
Comment (vgrebenschikov) [inline README.md]: s/TierConfigRequestAutomation/TierConfigAutomation/

logic:
Fulfillment - TierConfig
FulfillmentRequest - TierConfigRequest
FulfillmentAutomation - TierConfigAutomation

Comment (JaviCerveraIngram) [reply README.md]: Right. It will be fixed in the next commit.

**Comment by JaviCerveraIngram** [comment]
> 
> 
> Also, please make sure that code coverage is not decreased (better increased)

I'll write more tests for TierConfigAutomation to ensure coverage is increased.

**Comment by JaviCerveraIngram** [comment]
I'll reopen the PR when it meets the coverage requirements.

**Pull request merged**